test/extended/cli/basics.go

```diff
@@ -198,9 +198,19 @@ var _ = g.Describe("[sig-cli] oc basics", func() {
 			ok := rootCAs.AppendCertsFromPEM(kubeCA)
 			o.Expect(ok).To(o.BeTrue())
 
+			var certs []tls.Certificate
```

Comment:
Can you drop a comment here why this is needed for MicroShift?

Reply:
Done